apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -68,7 +76,16 @@ AprilTagDetector::~AprilTagDetector(){
   image_sub_.shutdown();
 }
 
+void AprilTagDetector::enableCb(const std_msgs::Bool& msg) {
+  enabled_ = msg.data;
```

Comment:
We should unsubscribe/subscribe when disabled.  Will avoid the unnecessary serialization of the topic when disabled.